### PR TITLE
Tickets/pipe2 d 65

### PR DESCRIPTION
**Pull request by ziggyman**

### CODE REVIEW

File: src/Spectra.cc [hunk 1]
```diff
@@ -945,7 +942,7 @@ namespace pfs { namespace drp { namespace stella { namespace math {
                                                                                                DispCorControl const& );
 
 }}}}
-
+
```
Comment (RobertLuptonTheGood) [inline src/Spectra.cc]: Please remove all pure white-space changes

File: tests/FiberTraces.py [hunk 2]
```diff
@@ -854,10 +794,12 @@ def suite():
 
     suites = []
     suites += unittest.makeSuite(FiberTraceTestCase)
-#    suites += unittest.makeSuite(tests.MemoryTestCase)
     return unittest.TestSuite(suites)
 
 def run(exit = False):
+    """Quiet down loggers which are too verbose"""
+    lsst.log.setLevel("CameraMapper", lsst.log.FATAL)
+    lsst.log.setLevel("afw.image.ExposureInfo", lsst.log.FATAL)
```
Comment (RobertLuptonTheGood) [inline tests/FiberTraces.py]: I'm a little suspicious of this one.   What are the unwanted messages, and did you file a ticket upstream to get rid of them?

In general we should not be setting log levels for other packages.

File: tests/Spectra.py [hunk 3]
```diff
@@ -451,6 +453,10 @@ def testWavelengthCalibrationWithRefSpec(self):
 
     def testWavelengthCalibrationWithoutRefSpec(self):
         if True:
+            logger = log.Log.getLogger("findAndTraceApertures")
+            logger.setLevel(log.WARN)
```
Comment (RobertLuptonTheGood) [inline tests/Spectra.py]: Why is this here, not in `setUpClass()`?

File: src/Spectra.cc (hunk 1 above)
Comment (ziggyman) [inline src/Spectra.cc]: done

File: tests/FiberTraces.py (hunk 2 above)
Comment (ziggyman) [inline tests/FiberTraces.py]: These are the outputs:

INFO  CameraMapper: Loading registry registry from /Users/azuri/stella-git/drp_stella_data/tests/data/PFS/registry.sqlite3
INFO  CameraMapper: Loading calibRegistry registry from /Users/azuri/stella-git/drp_stella_data/tests/data/PFS/CALIB/calibRegistry.sqlite3
INFO  afw.image.ExposureInfo: Empty WCS extension, using FITS header
INFO  afw.image.ExposureInfo: Empty WCS extension, using FITS header

I have not filed a ticket upstream yet but can do

File: tests/Spectra.py (hunk 3 above)
Comment (ziggyman) [inline tests/Spectra.py]: moved to setUp()

File: tests/FiberTraces.py (hunk 2 above)
Comment (PaulPrice) [inline tests/FiberTraces.py]: No need. Logging has changed a lot since LSST 12.1.

Comment (RobertLuptonTheGood) [inline tests/FiberTraces.py]: Also, why did you set the level to FATAL?  Was e.g. WARN not sufficient.

Paul: what are you proposing we do now?

Comment (PaulPrice) [inline tests/FiberTraces.py]: Upgrade our LSST code first, then re-evaluate the need for changing log levels. In the mean time, either live with the noise or suppress it, depending on how long you expect we're going to live in the current state.

Comment (PaulPrice) [inline tests/FiberTraces.py]: Let's leave this as it is until we upgrade LSST.
